app/models/profile.rb

```diff
@@ -1,12 +1,16 @@
 class Profile < ActiveRecord::Base
-  belongs_to :user
+  belongs_to :user, dependent: :destroy
   has_many :memberships
   has_many :organisations, through: :memberships
 
+  accepts_nested_attributes_for :user
+
   validates :name, :address, :postcode, :email, :tel, :mobile, presence: true
 
   validates :user_id, uniqueness: true, allow_nil: true
 
+  validates :email, uniqueness: true
```

Comment:
Does not validate that an email is formatted correctly. Something like the Devise Validatable module or 
https://github.com/plataformatec/devise/wiki/How-to:-Use-a-custom-email-validator-with-Devise